js/main.js

```diff
@@ -101,3 +133,15 @@ function viewSwap(view) {
     data.view = view;
   }
 }
+
+function handleEdit(entry) {
+  viewSwap('entryForm');
+  const $getData = entry.target.closest('li').getAttribute('data-entry-id');
+  data.editing = data.entries[data.entries.length - $getData];
```

Comment:
This won't always work. In the next issue you are going to be deleting elements. What if we create 20 entries, and then we delete 18 of them?. The `length` would now become 2 but the `entryId` never gets reset, so the next `entryId` would be 21.